src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
```

Comment:
Prefer using `const` for `totalCost` since it's not being reassigned, but rather its value is being updated in a functional manner.